report_pre_thesis/src/sections/introduction.tex

```diff
@@ -7,10 +7,16 @@ \section{Introduction}\label{sec:introduction}
 The primary interest are rocks that either formed in water or show indications of organic materials.\cite{chemcamNasaWebsite}
 
 ChemCam is a remote-sensing laser instrument developed by NASA in collaboration with the French national space agency (CNES).
-The instrument is used to gather Laser-Induced Breakdown Spectroscopy (LIBS) data from geological samples on Mars. The data itself consists of a series of spectral readings, each forming a spectrum. These spectra represent the emitted light from a plasma created when the laser interacts with the target sample. Captured over a range of wavelengths, each spectrum is composed of various emission lines. Each emission line is associated with a specific element, and its intensity reflects the concentration of that element in the sample. Consequently, the collection of spectra serves as a complex, multi-dimensional fingerprint of the elemental composition of the examined geological formations.
+The instrument is used to gather Laser-Induced Breakdown Spectroscopy (LIBS) data from geological samples on Mars.
+
+The laser pulses to ablate and remove any surface contaminants, such as dust and weathering layers, to expose the underlying material. The laser generates a plasma plume from the now-exposed sample material.
+This plasma plume emits light, and the data collected from this process consists of a series of spectral readings. Captured over a range of wavelengths, each spectrum is composed of various emission lines. Each emission line is associated with a specific element, and its intensity reflects the concentration of that element in the sample.
+Consequently, the collection of spectra serves as a complex, multi-dimensional fingerprint of the elemental composition of the examined geological formations.
 This data is then used to determine the elemental composition of these samples.\cite{cleggRecalibrationMarsScience2017}
 
-LIBS is a versatile analytical tool with broad applicability across various fields. In environmental monitoring, its spectral data are effectively used with machine learning and statistical methods like partial least squares and neural networks for detecting and quantifying soil pollutants. In industrial contexts, it is also utilized for quality control processes involving metals and alloys\cite{huang_progress_2023}.
+Due to its capability for remote analysis, LIBS enables processesing of materials without needing sample preparation. This feature allows for rapid analysis because of the immediate spectrum collection from the subsequent plasma. It does this while maintaining high spatial resolution due to its small observation footprints. This high resolution is essential for pinpointing and investigating small features. \cite{wiensChemcam2012}
```

Comment:
```suggestion
Due to its capability for remote analysis, LIBS enables processesing of materials without needing sample preparation. This enables rapid analysis because of the immediate spectrum collection from the subsequent plasma. It does this while maintaining high spatial resolution due to its small observation footprints. This high resolution is essential for pinpointing and investigating small features. \cite{wiensChemcam2012}
```